garage/tf/exploration_strategies/__init__.py

```diff
@@ -1,6 +1,8 @@
 """
 Exploration strategies used for RL algorithms.
 """
+from garage.tf.exploration_strategies.epsilon_greedy_strategy import (
+    EpsilonGreedyStrategy)
 from garage.tf.exploration_strategies.ou_strategy import OUStrategy
 
-__all__ = ["OUStrategy"]
+__all__ = ["OUStrategy", "EpsilonGreedyStrategy"]
```

Comment:
Does this need to be in alphabetical order

Reply:
Yes please do it in alphabetical order